src/BlockchainWriter/Router.ts

```diff
@@ -34,29 +34,23 @@ export class Router {
     const logger = this.logger.child({ method: 'onBlockchainWriterRequestTimestampRequest' })
 
     const messageContent = message.content.toString()
-    const { ipfsFileHashes, ipfsDirectoryHash } = JSON.parse(messageContent)
+    const { ipfsDirectoryHash } = JSON.parse(messageContent)
```

Comment:
@kennylavender The BlockchainWriter doesn't really need to know anything about individual claims now, just the batch